### PR TITLE
Add functionality #12

### PR DESCRIPTION
**Pull request by pavelpikta**
enable the option “Allow requests to the local network from hooks and services” in the “Outbound requests” section #12

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -103,6 +103,7 @@ gitlab_project_group: test_group
 gitlab_project_group_id: 1
 
 # gitlab webhooks
+gitlab_allow_local_requests: False
```
Comment (ViachaslauKabak) [inline defaults/main.yml:106]: Please set to True in molecule playbooks

File: README.md [hunk 2]
```diff
@@ -136,38 +136,40 @@ Requirements
...
      default: `Created API token`
-  - `gitlab_master_token` - user token; will be created if name not exists; does not visible for user; reset variable after creation; ***Should be set manually for versions earlier than 9.0***      
+  - `gitlab_master_token` - user token; will be created if name not exists; does not visible for user; reset variable after creation; ***Should be set manually for versions earlier than 9.0***
      default: `Token!234`
 # webhooks creation
-  - `gitlab_webhooks_create` - to create webhooks; correct gitlab_admin_token should be set if gitlab not installed by this role   
+  - `gitlab_webhooks_create` - to create webhooks; correct gitlab_admin_token should be set if gitlab not installed by this role
      default: `False`
   - `gitlab_webhooks_list` - map with webhooks parameters
+  - `gitlab_allow_local_requests` - enable the option [“Allow requests to the local network from hooks and services”](https://docs.gitlab.com/ee/security/webhooks.html) in the “Outbound requests” section
```
Comment (ViachaslauKabak) [inline README.md:171]: Is this gitlab option is available from particular version?

**Review by ViachaslauKabak — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (pavelpikta) [reply README.md:171]: starting with GitLab 10.6, all Webhook requests to the current GitLab instance server address and/or in a private network will be forbidden by default.
`( gitlab_version is version('10.6', '>=') or gitlab_version == "latest" )`

File: defaults/main.yml (hunk 1 above)
Comment (pavelpikta) [reply defaults/main.yml:106]: resolve in e2e411f

**Review by tgadiev — APPROVED** [review]

**Review by ViachaslauKabak — APPROVED** [review]

**Pull request merged**